validphys2/src/validphys/filters.py

```diff
@@ -110,7 +111,11 @@ def filter_closure_data(filter_path, data, fakepdf, fakenoise, filterseed):
 
 
 def filter_closure_data_by_experiment(
-    filter_path, experiments_data, fakepdf, fakenoise, filterseed, experiments_index
+    filter_path, experiments_data, fakepdf, fakenoise, filterseed, experiments_index,
```

Comment:
Can you add some docs about the new arguments?  (i.e. how they should be used and what they are and so on ...)